docs/migrating-from-0.9.md

```diff
@@ -0,0 +1,251 @@
+# Migrating from 0.9
+
+Genkit 1.0 introduces some of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.9, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
+If you're using 0.5, use <> to upgrade to 0.9 first before upgrading to 1.0
+
+
+## Streaming functions do not require an await
+
+
+**Old:**
+
+```ts
+const { stream, response } = await ai.generateStream(`hi`);
+const { stream, output } = await myflow.stream(`hi`);
+```
+
+**New**
+
+```ts
+const { stream, response } = ai.generateStream(`hi`);
+const { stream, output } = myflow.stream(`hi`);
+```
+
+
+## Beta APIs
+
+We're introducing an unstable, Beta API channel, and leaving session, chat and Genkit client APIs in beta as we continue to refine them. 
+
+Note: When using the APIs as part of the Beta API, you may experience breaking changes outside of SemVer. Breaking chanhes may occur on minor release (we'll try to avoid making breaking beta API changes on patch releases).
+
+**Old:**
+
+```ts
+import { genkit } from 'genkit';
+const ai = genkit({...})
+const session = ai.createSession({ ... })
+```
+
+**New**
+
+```ts
+import { genkit } from 'genkit/beta';
+const ai = genkit({...})
+const session = ai.createSession({ ... })
+```
+
+**Old:**
+
+```ts
+import { runFlow, streamFlow } from 'genkit/client';
+```
+
+**New**
+
+```ts
+import { runFlow, streamFlow } from 'genkit/beta/client';
+```
+
+## Introducing new `@genkit-ai/express` package
+
+The package contains utilities to make it easier to build an express app using Express.js.
+
+Refer to https://js.api.genkit.dev/modules/_genkit-ai_express.html for more details.
```

Comment:
I'd expect this to go to a github doc as the primary source

Reply:
we'll try to document more an more things in api reference...

Reply:
it'll be a bit confusing to have 3 sources of truth, the API reference, the github repo and the Firebase dcos

Reply:
By "the GitHub repo" do you mean the samples there or the docs in GitHub?

Reply:
The docs in GitHub

Reply:
we want to prompte https://js.api.genkit.dev, I'm going to leave it for now.